util/src/main/java/io/camunda/zeebe/util/FatalErrorHandler.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.util;
+
+import java.lang.Thread.UncaughtExceptionHandler;
+import org.slf4j.Logger;
+
+public final class FatalErrorHandler implements UncaughtExceptionHandler {
```

Comment:
:x: Let's document the behavior as any thing that just "exits" is probably counter intuitive.

Reply:
Let's also document what is currently considered a fatal error and why, I can see that getting lost over time :smile: